sdk/ios/Configsum/Configsum/Configsum/Inputs/Context.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Context.swift
+//  Configsum
+//
+//  Created by Alexandru Savu on 2017-10-10.
+
+public enum Platform: String, Codable {
+    case android
+    case iOS
+    case watchOS
+}
+
+public class Context: Codable {
```

Comment:
Isn't it possible to make this a struct?

Reply:
not if we're gonna use it in obj-c. which reminds me to add @objc to the public interface